CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Change Log - @itwin/insights-client
 
+## 0.5.3
```

Comment:
Adding a peerDependency is a breaking change. New version should be 0.6.0

https://github.com/semver/semver/issues/502